packages/core-components/src/components/ProgressBars/Gauge.tsx

```diff
@@ -104,10 +115,12 @@ export const getProgressColor: GaugePropsGetColor = ({
  */
 
 export function Gauge(props: GaugeProps) {
+  const hoverRef = useRef() as RefObject<HTMLInputElement>;
```

Comment:
```suggestion
  const hoverRef = useRef<HTMLElement>();
```

or possibly `HTMLElement | undefined`